### DISCUSSION TITLE
Should OpynDAO be created？

### DISCUSSION THREAD

**Discussion by wozainalea** [opynfinance/OGP#24] [Q&A]
When we work in the crypto market, we often encounter multiple choices. Many of us have our own ideas. DAO can solve this problem by classifying and voting on various people's thinking.  So does the team have the idea and plan to build a DAO?

**Comment by travel-subsidize0stature-resident** [comment]
> When we work in the crypto market, we often encounter multiple choices. Many of us have our own ideas. DAO can solve this problem by classifying and voting on various people's thinking. So does the team have the idea and plan to build a DAO?

Check this out: https://discord.com/channels/590664003815211058/829019810154807307/829025628215836712

<img width="500" alt="image" src="https://user-images.githubusercontent.com/94918584/143659524-6b507a56-5eea-48b2-884d-68480d0ffd49.png">
